aws-ssm-patchbaseline/aws-ssm-patchbaseline.json

```diff
@@ -1,6 +1,6 @@
 {
     "typeName": "AWS::SSM::PatchBaseline",
-    "description": "Resource Type definition for AWS::SSM::PatchBaseline",
+    "description": "AMAZON_LINUX",
```

Comment:
why do you change the description AMAZON_LINUX, this should apply to all OS

Reply:
Modified the Description for as meaningful for AWS OS flavors